### DISCUSSION TITLE
Temporarily Exclude DTMF Library

### DISCUSSION THREAD

**Discussion by eopeter** [eopeter/flutter_dialpad#16] [General]
@cybex-dev you said "Regarding `flutter_dtmf`, it seems Web has a bit of a problem when including the library - you may have noticed it's disabled temporarily until a fix has been added."

**Comment by eopeter** [comment]
Can we exclude it just for web to publish the latest version or wait for the fix?

**Comment by cybex-dev** [comment]
Hi @eopeter, I believe we can though it seems simply including the `flutter_dtml` package in `pubspec.yaml` seems to cause this issue (atleast on Web).

**Comment by cybex-dev** [comment]
Further, it seems there is also another issue with regards to grabbing the [AudioContext window object](https://github.com/eopeter/flutter_dtmf/blob/master/lib/dtmf_web.dart#L63), the error I do not recall but has been a bit of a challenge to resolved.

I'll be looking at resolving this weekend for a release next week.
